mediaflow_proxy/extractors/dlhd.py

```
@@ -2,259 +2,254 @@
-        
-        # Apply newkso.ru specific headers if the URL contains newkso.ru
-        final_headers = self._get_headers_for_url(url, request_headers)
-        
-        return await super()._make_request(url, method, final_headers, **kwargs)
+    async def _make_request(self, url: str, method: str = "GET", headers: Optional[Dict] = None, **kwargs) -> Any:
+        """Override to disable SSL verification for this extractor and use fetch_with_retry if available."""
+        from mediaflow_proxy.utils.http_utils import create_httpx_client, fetch_with_retry
+
+        timeout = kwargs.pop("timeout", 15)
+        retries = kwargs.pop("retries", 3)
+        backoff_factor = kwargs.pop("backoff_factor", 0.5)
```

Ruff (0.13.1)

Local variable `backoff_factor` is assigned to but never used

Remove assignment to unused variable `backoff_factor`

(F841)